server/routes/grafana/tasks.py

```
@@ -180,274 +216,241 @@ def process_grafana_alert(
+                                    handle_correlated_alert(
+                                        cursor=cursor, user_id=user_id,
+                                        incident_id=correlation_result.incident_id,
+                                        source_type="grafana", source_alert_id=per_alert_source_id,
+                                        alert_title=alert_title, alert_service=service,
+                                        alert_severity=severity,
+                                        correlation_result=correlation_result,
+                                        alert_metadata=alert_metadata, raw_payload=alert_payload,
+                                    )
+                                    conn.commit()
+                                    continue
+                            except Exception as corr_exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)